WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -36,6 +37,9 @@ public BitcoinTabSettingsViewModel(IApplicationSettings settings)
 
 		_bitcoinP2PEndPoint = settings.BitcoinP2PEndPoint;
 		_dustThreshold = settings.DustThreshold;
+
+		this.WhenAnyValue(x => x.Settings.BitcoinP2PEndPoint)
+			.Subscribe(x => BitcoinP2PEndPoint = x);
```

Comment:
Why isn't this change also required for `DustTreshold`?

Reply:
Ok I had the doubt because its presence in `ApplicationsSettings.ApplyChanges` but I see that it cannot cause an issue